packages/odyssey/src/scss/abstracts/_tokens.scss

```diff
@@ -58,7 +58,7 @@ $focus-ring-danger: $color-danger-light;
 $text-body: cv('gray', '900');
 $text-heading: cv('gray', '900');
 $text-sub: cv('gray', '600');
-$text-code: cv('gray', '900');
+$text-code: #640019;
```

Comment:
This should get into the color map before merge.

Reply:
Im reverting this change, I do think we need to differentiate code color but its out of scope for this PR.